src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,10 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public abstract boolean isMutating( );
+    /**
+     * Checks whether the commands change the data
```

Comment:
A more precise phrasing would be `Returns true if.....`

It is better to mention that it returns true if the command *potentially* changes the data. Try to be as precise as possible.